modules/end-user-guide/pages/troubleshooting-webview-loading-error.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:description: Troubleshooting Webview loading error
+:keywords: user-guide, troubleshooting-webview-loading-error
+:navtitle: Troubleshooting Webview loading error
+:page-aliases: .:troubleshooting-webview-loading-error.adoc
+
+[id="troubleshooting-webview-loading-error"]
+
+= Troubleshooting Webview loading error
+
+pass:[<!-- vale RedHat.Spelling = NO -->]
+
+If you use Microsoft Visual Studio Code - Open Source in an incognito or private browser window, you can encounter an issue with Webviews. The Webview is not loaded and an error message appears: *Error loading webview: Error: Could not register service workers*.
+
+The issue affects the following browsers:
+
+* Google Chrome in Incognito mode
+* Mozilla Firefox in Private Browsing mode
```

Comment:
If the error might happen on any other browser in incognito mode, then it might be worth to consider removing this list if the SME agrees.
There are incognito modes in other browsers:
* https://blogs.opera.com/news/2014/10/how-to-open-private-window-opera-for-computers/
* https://support.microsoft.com/en-us/microsoft-edge/browse-inprivate-in-microsoft-edge-e6f47704-340c-7d4f-b00d-d0cf35aa1fcc
What does the SME think?

Reply:
We just faced the problem in these two browsers. We didn't check it for other browsers.
That's the reason why the specific browsers are mentioned.

Reply:
I see your point @max-cx. Thanks for the clarification.
I agree - let's remove the browsers' names.

Reply:
I updated the wording, please let me know if it's ok with you now.